### ISSUE TITLE
Link names do not have discernible text

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/pulmap#1002]
### Errors in spec/features/accessibility_spec.rb

1) accessibility when visiting main page complies with WCAG

       link-name: Links must have discernible text (serious)
       https://dequeuniversity.com/rules/axe/4.3/link-name?application=axeAPI
       The following 1 node violate this rule:
           
               Selector: .leaflet-bar-part
               HTML: <a class="leaflet-bar-part leaflet-bar-part-single" title="" href="#" data-toggle="tooltip" data-original-title="Search for location"></a>
               Fix all of the following:
               - Element is in tab order and does not have accessible text
               Fix any of the following:
               - Element does not have text that is visible to screen readers
               - aria-label attribute does not exist or is empty
               - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
               - Element has an empty title attribute

2) accessibility when visiting search results page complies with WCAG

       link-name: Links must have discernible text (serious)
       https://dequeuniversity.com/rules/axe/4.3/link-name?application=axeAPI
       The following 2 nodes violate this rule:
           
               Selector: .remove
               HTML: <a class="btn btn-default btn-constraint remove" href="/?q="><span class="remove-icon"></span><span class="d-none">Remove constraint Source: princeton-1r66j405w</span></a>
               Fix all of the following:
               - Element is in tab order and does not have accessible text
               Fix any of the following:
               - Element does not have text that is visible to screen readers
               - aria-label attribute does not exist or is empty
               - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
               - Element has no title attribute
               
               Selector: .leaflet-bar-part
               HTML: <a class="leaflet-bar-part leaflet-bar-part-single" title="" href="#" data-toggle="tooltip" data-original-title="Search for location"></a>
               Fix all of the following:
               - Element is in tab order and does not have accessible text
               Fix any of the following:
               - Element does not have text that is visible to screen readers
               - aria-label attribute does not exist or is empty
               - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
               - Element has an empty title attribute
